docs/journeys-from-trip.md

```diff
@@ -0,0 +1,45 @@
+# `journeysFromTrip(tripId, previousStopover, to, [opt])`
```

Comment:
Just for my understanding: Is the `previousStopover` always ignored internally and overwritten by the last one that was passed, anyway, or does it make a difference? (I'm asking because you write below that the DB Navigator always passes the first one, but AFAIK the navigator then shows results that indicate the server picked the last-passed one anyway)

Reply:
I don't really know. 🤷